metrics/deploy/docs/diagrams/deploy_sequence.puml

```diff
@@ -0,0 +1,50 @@
+@startuml deploy_sequence
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant Terminal
+
+box CLI
+participant DeployCommand
+participant FirebaseCommand
+participant FlutterCommand
+participant GCloudCommand
+endbox
+
+Bob -> Terminal : enters 'build/metrics deploy'
+
+activate Terminal
+Terminal -> DeployCommand : run()
+
+activate DeployCommand
+
+DeployCommand -> GCloudCommand : login()
```

Comment:
It is not clear how login happens here and how `GCloudCommand` performs it. As I can see, `GCloudCommand` interacts with `gcloud` installed and runs `gcloud auth login` that is interactive. So does it make sense to notice in this diagram that `Bob` interacts with CLI more than once? I would suggest creating a separate box for each standalone CLI tool the `deploy` one uses and then clearly notice that these tools are used and somewhere `Bob` interacts with them. I suppose it is really important in understanding the current implementation and what we should do in the future to automate it as much as possible.